cmd/influx/write.go

```diff
@@ -74,41 +90,102 @@ func cmdWrite(f *globalFlags, opt genericCLIOpts) *cobra.Command {
 	return cmd
 }
 
+func (writeFlags *writeFlagsType) dump(args []string) {
+	if writeFlags.Debug {
```

Comment:
don't we have a `log.Debug` function that would be better here?

Reply:
The built-in golang's log package has no Debug fn, OOTB it prints messages to stderr.